src/arc.ts

```diff
@@ -124,9 +125,35 @@ function getBiconomy(provider: any): any {
   return biconomy;
 }
 
+
+/**
+ * Function to show user contract wallet address to inform him that he/she
+ * should make sure to have reputation in this address in order to vote for dao
+ * without paying transaction fee.
+ * @param userContract User contract wallet address
+ */
+function showUserContractAddress(userContract: string, options?: IEnableWalletProviderParams) {
+  if (userContract) {
+    // eslint-disable-next-line no-console
+    console.info(`User contract wallet ${userContract}`);
+    if (options)
+      showSimpleMessage(
+        {
+          title: "Your Contract Wallet",
+          body: React.createElement('span', null,
+            React.createElement('div', null, `Your contract wallet address is: ${userContract}.`),
+            React.createElement('div', null, `So that you can vote without paying a transaction fee, make sure you have reputation in this address.`))
+        });
+  }
+}
+
 function showUserNotification(message: string, options?: IEnableWalletProviderParams, type?: NotificationStatus) {
-  if (options && options.showNotification && type)
+  if (options?.showNotification && type) {
```

Comment:
Is this ? a typo here?

Reply:
is not a typo.  if options is undefined or null, showNotification will evaluate to undefined.